bin/lib/getStorybookInfo.js

```diff
@@ -23,59 +20,36 @@ const viewLayers = {
   '@storybook/svelte': 'svelte',
   '@storybook/preact': 'preact',
   '@storybook/rax': 'rax',
-  '@web/dev-server-storybook': 'web-components',
 };
 
-const supportedAddons = {
-  '@storybook/addon-a11y': 'a11y',
-  '@storybook/addon-actions': 'actions',
-  '@storybook/addon-backgrounds': 'backgrounds',
-  '@storybook/addon-centered': 'centered',
-  '@storybook/addon-contexts': 'contexts',
-  '@storybook/addon-cssresources': 'cssresources',
-  '@storybook/addon-design-assets': 'design-assets',
-  '@storybook/addon-docs': 'docs',
-  '@storybook/addon-essentials': 'essentials',
-  '@storybook/addon-events': 'events',
-  '@storybook/addon-google-analytics': 'google-analytics',
-  '@storybook/addon-graphql': 'graphql',
-  '@storybook/addon-info': 'info',
-  '@storybook/addon-jest': 'jest',
-  '@storybook/addon-knobs': 'knobs',
-  '@storybook/addon-links': 'links',
-  '@storybook/addon-notes': 'notes',
-  '@storybook/addon-ondevice-actions': 'ondevice-actions',
-  '@storybook/addon-ondevice-backgrounds': 'ondevice-backgrounds',
-  '@storybook/addon-ondevice-knobs': 'ondevice-knobs',
-  '@storybook/addon-ondevice-notes': 'ondevice-notes',
-  '@storybook/addon-options': 'options',
-  '@storybook/addon-queryparams': 'queryparams',
-  '@storybook/addon-storyshots': 'storyshots',
-  '@storybook/addon-storysource': 'storysource',
-  '@storybook/addon-viewport': 'viewport',
-};
+// Double inversion on Promise.all means fulfilling with the first fulfilled promise, or rejecting
+// when _everything_ rejects. This is different from Promise.race, which immediately rejects on the
+// first rejection.
+const invert = (promise) => new Promise((resolve, reject) => promise.then(reject, resolve));
+const raceFulfilled = (promises) => invert(Promise.all(promises.map(invert)).then((arr) => arr[0]));
```

Comment:
Lol neat trick

Reply:
It was nice to flex my async muscles again.